Examples/Scotty-tvOSExample/Routing/URL+RouteConvertible.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  URL+RouteConvertible.swift
+//  Scotty-iOS
```

Comment:
We will want to make sure the comment in the headers are uniform - Scotty-tvOSExample